src/main/resources/crafter/studio/database/createDDL.sql

```diff
@@ -246,25 +246,22 @@ CREATE TABLE IF NOT EXISTS `group`
   ROW_FORMAT = DYNAMIC ;
 
 INSERT IGNORE INTO `group` (id, record_last_updated, org_id, group_name, group_description)
-VALUES (1, CURRENT_TIMESTAMP, 1, 'crafter-admin', 'Crafter Administrator group') ;
+VALUES (1, CURRENT_TIMESTAMP, 1, 'org-admin', 'Organization Administrator group') ;
```

Comment:
2 things:
- must have consistency, use _ not -
- isn't this the global super admin group part of org 1, and therefore should be more than org_admin?